app/src/components/Courses/ToolTip.tsx

```diff
@@ -0,0 +1,73 @@
+// Icon from: https://materialdesignicons.com/
+
+import useSWR from 'swr'
+
+import Icon from '@mdi/react';
+import { mdiWindowClose } from '@mdi/js';
+
+interface Inputs {
+    textContent: string,
+    myIndex: number,
+    maxIndex: number,
+    callBack: Function
+}
+
+
+
+// ToolTip component
+export const ToolTip = ({textContent,myIndex,maxIndex, callBack}:Inputs) => {
```

Comment:
It would probably be a good idea to move the textContent to be in the children prop instead. This makes it a little more readable when using it, and it also allows for customisation of the text itself.